Source/ACE/Network/NetworkSession.cs

```diff
@@ -168,24 +181,22 @@ private void FlushBundle()
 

```

Comment:
FlushBundle, above, needs a lock but GitHub won't let me comment on it since you didn't change it.  It's what actually wipes out the bundle and resets it.  If this is being called within a lock, then it really should be doc'ed to state that the lock is a requirement for calling it.  If it's only being called in one place, just move the code to where it's called in the lock.